net/rpc/server_test.go

```diff
@@ -436,6 +450,7 @@ func (codec *CodecEmulator) Call(serviceMethod string, args *Args, reply *Reply)
 func (codec *CodecEmulator) ReadRequestHeader(req *Request) error {
 	req.ServiceMethod = codec.serviceMethod
 	req.Seq = 0
+	req.SourceAddr = &net.TCPAddr{IP: []byte{99, 99, 99, 99}, Port: 8080, Zone: ""}
```

Comment:
You may want to consider using the [`TCPAddrFromAddrPort`](https://pkg.go.dev/net#TCPAddrFromAddrPort) and [`MustParseAddrPort`](https://pkg.go.dev/net/netip#MustParseAddrPort) methods from the standard library here. It doesn't save you any characters, but does save you having to manually construct a byte-slice and lets you use a more familiar `host:port` syntax 😄

```suggestion
	req.SourceAddr = net.TCPAddrFromAddrPort(netip.MustParseAddrPort("1.2.3.4:8080"))
```

Reply:
Bumped to use go 1.19 since these APIs were introduced in go 1.18 and module currently uses go 1.17.